app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -665,6 +678,51 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         startActivityForResult(speechIntent, VoiceSearchActivity.SPEECH_REQUEST_CODE)
     }
 
+    private fun addQrButton(searchFragmentState: SearchFragmentState) {
+        if (qrButtonAlreadyAdded) return
+        val searchEngine = searchFragmentState.searchEngineSource.searchEngine
+
+        val isVisible =
+            searchEngine?.id?.contains("google") == true &&
+                isSpeechAvailable() &&
+                requireContext().settings().shouldShowVoiceSearch
+
+        if (isVisible) {
+            toolbarView.view.addEditActionEnd(
+                BrowserToolbar.Button(
+                    AppCompatResources.getDrawable(requireContext(), R.drawable.ic_qr)!!,
+                    requireContext().getString(R.string.search_scan_button),
+                    visible = { true },
```

Comment:
This isn't necessary since it's already a default value

Reply:
Ah, I'll remove it.